apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -383,7 +414,8 @@ const aichatSlice = createSlice({
       // and start a new session, see clearChatMessages.
       if (
         messageToRemovePosition < 0 ||
-        updatedMessages[messageToRemovePosition].role !== Role.MODEL_UPDATE
+        (updatedMessages[messageToRemovePosition].role !== Role.MODEL_UPDATE &&
```

Comment:
I think we should probably update the comment above and rename this reducer now that we have these error messages